components/classes/digital-asset.example.1.json

```diff
@@ -7,7 +7,17 @@
     "xdm:name": "jump.png",
     "xdm:description": "A man is performing a mid-air jump on a skateboard at an indoor skate park",
     "xdm:assetType": "image",
-    "xdm:thumbnailURL": "https://domain/thumbnail.jpeg"
+    "xdm:thumbnailURL": "https://domain/thumbnail.jpeg",
+    "xdm:additionalIDs": [
+      {
+        "xdm:platform": "ACA",
```

Comment:
don't know if we want to mess with enums here for the `platform` value? fine by me not to, but probably more correct to have as enum

Reply:
We discussed this in our teams meeting